controllers/login.js

```diff
@@ -1,16 +1,16 @@
-const {authenticateLogin} = require('../models/loginDbModel')
-const {createNewJwt} = require('../services/authentication')
+const { authenticateLogin } = require('../models/loginDbModel')
+const { createNewJwt } = require('../services/authentication')
 
 exports.login = async (req, res) => {
-    try {
-        const {username, password} = req.body
-        if (await authenticateLogin(username, password)) {
-            const newToken = createNewJwt(username)
-            return res.status(200).send(newToken)
-        } else {
-            return res.status(400).send('Login failed, username or password incorrect')
-        }
-    } catch (err) {
-        return res.status(400).send(`Login failed, ${err.message}`)
+  try {
+    const { username, password } = req.body
+    if (await authenticateLogin(username, password)) {
+      const newToken = createNewJwt(username)
+      return res.status(200).send(newToken)
+    } else {
+      return res.status(400).send('Login failed, username or password incorrect')
```

Comment:
why 400? 400 is bad request, this is not the case here. just do:
` return next(new Error('Login failed, username or password incorrect'))`